src/test/java/org/jenkinsci/plugins/gitclient/GitClientCliCloneTest.java

```diff
@@ -155,7 +157,8 @@ public void test_checkout_interrupted() throws Exception {
         assertThat("Lock file removed by checkout", lockFile, is(not(aReadableFile())));
     }
 
-    @Issue("JENKINS-25353")
+    @Test
+	@Issue("JENKINS-25353")
```

Comment:
Same as preceding comment
```suggestion
    @Issue("JENKINS-25353")
```